### ISSUE TITLE
Invalid Weekday Number

### ISSUE THREAD

**Issue by eric-corumdigital** [SpiritIT/timezonecomplete#43]
```
DateTime {
  kind: 'DateTime',
  _zone:
   TimeZone {
     classKind: 'TimeZone',
     _name: 'Africa/Abidjan',
     _dst: true,
     _kind: 2 },
  _zoneDate:
   TimeStruct {
     _components:
      { year: -138909,
        month: 12,
        day: 31,
        hour: 23,
        minute: 59,
        second: 59,
        milli: 999 },
     _unixMillis: -4445680291200001 } }
```

The weekday number (given by `weekDay()`) is `-3` and should be `4` for Thursday.

```
DateTime {
  kind: 'DateTime',
  _zone:
   TimeZone {
     classKind: 'TimeZone',
     _name: 'Africa/Abidjan',
     _dst: true,
     _kind: 2 },
  _zoneDate:
   TimeStruct {
     _components:
      { year: -206790,
        month: 12,
        day: 31,
        hour: 23,
        minute: 59,
        second: 59,
        milli: 999 },
     _unixMillis: -6587797737600001 } }
```

The weekday number for this is `-2` and should be `5` for Friday.

From this, I extrapolate a correction is to add 7 to negative weekdays.

**Comment by rogierschouten** [comment]
whoops...thank you for reporting this!

**Comment by rogierschouten** [comment]
please note that the IANA time zone database is not accurate before 1970 so you may not want to use any currently available library on NPM for that purpose (moment.js etc are all based on IANA and it won't work.   

I will start working on a fix for the bug though

**Comment by rogierschouten** [comment]
Fix released as 5.11.1
